ns/1.0/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<head>
+<title>CapiTainS Metadata Element Set, Version 1.0</title>
+<style>
+h1 { color: #F60; font: bold 130% Verdana, Arial, "Arial Unicode MS", Helvetica, sans-serif; }
+h2 { color: #000; font: bold 125% Verdana, Arial, "Arial Unicode MS", Helvetica, sans-serif; }
+h3 { color: #000; font: bold 115% Verdana, Arial, "Arial Unicode MS", Helvetica, sans-serif; }
+h4 { color: #666; font: bold 100% Verdana, Arial, "Arial Unicode MS", Helvetica, sans-serif; font-style: italic; }
+h5 { color: #777; font: bold 80% Tahoma, Geneva, "Arial Unicode MS", Arial, Helvetica, sans-serif; }
+h6 { color: #000; font: normal 100% Verdana, Arial, "Arial Unicode MS", Helvetica, sans-serif; }
+
+code, pre { background: #EEE; line-height: 140%; }
+
+table.docinfo { background: #FFC; width: 100%; margin-bottom: 10px; }
+table.docinfo th { text-align: right; padding: 4px; width: 15%; }
+table.docinfo td { text-align: left; padding: 4px; width: 85%; }
+
+table.border { border-top: 1px solid #CCC; border-left: 1px solid #CCC; margin-bottom: 10px; }
+table.border th { text-align: left; background: #EEE; font-family: Verdana, Arial, "Arial Unicode MS", Helvetica, sans-serif; }
+table.border td, table.border th { border-right: 1px solid #CCC; border-bottom: 1px solid #CCC; padding: .5em; }
+</style>
+</head>
+<body>
+<a name="maincontent" id="maincontent"></a>
+
+<h1>CapiTainS Metadata Element Set, Version 1.0</h1>
+<table cellspacing="0" class="docinfo">
+<tr>
+<th>Identifier:</th>
+<td><a href="http://purl.org/capitains/ns/1.0#">http://purl.org/capitains/ns/1.0#</a></td></tr>
+<tr>
+<th>Date Issued:</th>
+<td>2017-04-27</td></tr>
+<tr>
+<th>Status of document:</th>
+<td>????</td></tr>
+<tr>
+<th>Description of document:</th>
+<td>This document is a quick reference for the CapiTainS Metadata Element Set, Version 1.0.
+</td></tr>
+</table>
+<a name="introduction" id="introduction"></a> 
+<h2>Introduction</h2>
+
+<p>The following elements have been added by CapiTainS to the normal CTS element set. Theses elements should only be used in the context of the CapiTainS guidelines <a href="http://capitains.org/pages/guidelines">http://capitains.org/pages/guidelines</a>.</p>
+
+<h4>References</h4>
```

Comment:
I completely removed the references because their purpose on the DC page is to explain the references that they have later in the document. Since we have none, I have commented them out (so that we can add references in later if we need to).